squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/mime/MimeTypeDetector.java

```diff
@@ -8,35 +8,79 @@
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.ListIterator;
 
+/**
+ * Standard implementation of the {@link TypeDetector} interface.
+ *
+ * @author Abhishek Hassan Chandrashekar (abhihc@mail.uni-paderborn.de).
+ */
 public class MimeTypeDetector implements TypeDetector {
+
     private static final Logger LOGGER = LoggerFactory.getLogger(RDFAnalyzer.class);
 
+    protected List<Lang> mimeTypes = new ArrayList<>();
+
     public Lang detectMimeType(File data) {
```

Comment:
please add javadoc comments to methods (unless the method is only a getter or setter method)

Reply:
I have added the javadoc comments